packages/components/src/border-box-control/border-box-control/README.md

```diff
@@ -116,6 +116,24 @@ _Note: The will be `undefined` if a user clears all borders._
 
 - Required: Yes
 
+### `popoverClassNames`: `Object`
+
+An object defining CSS classnames for all the inner `BorderControl` popover
+content.
```

Comment:
Totally optional, but we could potentially add a sentence like the following to explain why someone might want to pass in classnames for each of the popovers (though it's already pretty clear from the implicit usage in the other PR):

"By default, popovers are displayed relative to the button that initiated the popover. By supplying classnames for each individual popover, it is possible to add styling rules to align the popover positions to an unrelated design element, for example, the sidebar inspector in the block editor."

Reply:
👍  Thanks for the suggestion. I've added the explanatory comment in https://github.com/WordPress/gutenberg/pull/39753/commits/b76db0096d7dc1a320bed9f3bcca32a2b9f2a6b4